app/helpers/link_to_helper.rb

```diff
@@ -254,6 +254,20 @@ def unhappy_url(info_request = nil)
     end
   end
 
+  # Google Analytics
+
+  # if used, value must an integer
+  def analytics_event_code(category, action, label=nil, value=nil, label_is_script=false)
```

Comment:
I'd name the required arguments `event_category` and `event_action` so that they map to the titles used [by Google](https://github.com/mysociety/alaveteli/pull/3110#issue-141910287)